pkg/koyeb/instances_exec.go

```diff
@@ -91,7 +91,17 @@ func (e *Executor) Run(ctx context.Context) (int, error) {
 	}
 	go closeOn(c, syscall.SIGINT, syscall.SIGTERM)
 
-	pushErrCh := e.push(ctx, c, e.stdin)
+	r := &ApiExecCommandRequest{
+		Id:   &e.instanceId,
+		Body: koyeb.NewExecCommandRequestBody(),
+	}
+	r.Body.SetCommand(e.cmd)
+	err = e.pushOne(ctx, c, r)
+	if err != nil {
+		return -1, errors.Wrap(err, "could not intialize RPC with server")
```

Comment:
Sorry a bit late, but you should return a zero value for the int since the error takes precedence over the returned value.